dockstore-webservice/src/main/java/io/dockstore/webservice/core/Organization.java

```diff
@@ -98,6 +98,11 @@
     @UpdateTimestamp
     private Timestamp dbUpdateDate;
 
+    @Column
+    @Pattern(regexp = "(.*)(?i)(\\.jpg|\\.jpeg|\\.png)")
```

Comment:
Does gravatar support .gif? If so, perhaps add it.

Reply:
it does support .gif

Reply:
Now allows .gif in newest commit. Will change in UI